### DISCUSSION TITLE
Phidgets stepper controller support?

### DISCUSSION THREAD

**Discussion by Peter-Patter** [artisan-roaster-scope/artisan#891] [Q&A] [answered]
Hi everyone, 

I've been searching the internet for weeks for an answer to this question. I know that Artisan works with Phidgets DC and linear actuator controllers, but what I want to know is will it work with Phidgets stepper motor controllers, like [this one ](https://www.robotshop.com/uk/phidget-vint-4a-stepper-motor-controller.html?gclid=Cj0KCQjw-daUBhCIARIsALbkjSaM26iSEUbP_3V4RN-LlgdiGd1cJldizbfzHo-yj0Pvq9-RFGo28IcaAny1EALw_wcB), there's also a 2A version. 
![image](https://user-images.githubusercontent.com/106607061/171202892-fb02fe4d-e6ad-41a0-ae92-2d6900323fa3.png)

I'm toying with the idea of controlling two sliding fan dampers with small stepper motors. I don't have much experience but from my research it looks like stepper motors would offer the precision required to move the dampers in and out in predetermined increments, i'd use a strip like this attached to the sliders
![image](https://user-images.githubusercontent.com/106607061/171201939-b4c2d714-d08e-4c76-bd6d-e32c35e7d7ca.png)
with a corresponding cog around the shaft of the stepper motors. I already have a Phidget VINT hub so I thought hooking up a the stepper motor controller would be the simplest solution, but would love some advice from people with more knowledge on this. I don't know if the events options in Artisan has support for the control of these type of Phidgets devices. 

Thank you.

**Comment by MAKOMO** [comment, marked as answer]
Sorry, Artisan is currently not supporting the Phidgets Stepper API which is used to control stepper controllers.

**Reply by Peter-Patter** [reply]
Thanks Marko, not the answer I was hoping for but an answer nonetheless. Back to the drawing board, looks like linear actuators are the way forward then.
